src/App.tsx

```diff
@@ -55,7 +55,10 @@ function App() {
         <h2>Trend</h2>
         <div className='app-box'>
           {isLoading ? 'loading.....' : data.map((element) =>
-            RenderCards(true,element)
+            <RenderCards
+              showClassic={true}
```

Comment:
Quand c'est une props de type boolean tu peux écrire ça en raccourci
```suggestion
              showClassic
```